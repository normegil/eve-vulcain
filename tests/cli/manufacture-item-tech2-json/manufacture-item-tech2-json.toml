bin.name = "eve-vulcain"
args = "manufacture item \"Damage Control II\" --strict --json -vvv"
status.code = 0
