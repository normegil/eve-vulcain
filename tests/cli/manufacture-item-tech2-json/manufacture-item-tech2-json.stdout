{"searched_item_name":"Damage Control II","markets":[{"name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","regional_average_volumes":10822,"lowest_price":122700.0}],"invention":{"base_cost_run":382398.28969963355,"success_chance":0.44200000000000006,"runs":10,"total_run":86515.4501582881},"facilities":[{"facility_name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","costs":{"inputs":{"inputs":[{"name":"Oscillator Capacitor Unit","orders":[{"quantity":1,"price_per_unit":48191.09,"total":48191.09}]},{"name":"Morphite","orders":[{"quantity":1,"price_per_unit":43482.24,"total":43482.24}]},{"name":"Mechanical Parts","orders":[{"quantity":5,"price_per_unit":12060.16,"total":60300.8}]},{"name":"Plasma Pulse Generator","orders":[{"quantity":1,"price_per_unit":19833.91,"total":19833.91}]},{"name":"R.A.M.- Armor/Hull Tech","orders":[{"quantity":1,"price_per_unit":1060.36,"total":1060.36}]},{"name":"Construction Blocks","orders":[{"quantity":3,"price_per_unit":12334.49,"total":37003.47}]},{"name":"Fernite Carbide Composite Armor Plate","orders":[{"quantity":1,"price_per_unit":9372.52,"total":9372.52}]},{"name":"Quantum Microprocessor","orders":[{"quantity":1,"price_per_unit":65805.53,"total":65805.53}]},{"name":"Damage Control I","orders":[{"quantity":1,"price_per_unit":8207.74,"total":8207.74}]}],"total":293257.66},"job_cost":{"estimated_item_value":177538.88791922803,"system_cost_index":0.2088,"facility_tax":0.0025,"total":37780.275349211726,"job_cost_modifier":null},"total_run":417553.3855074998,"total_per_unit":417553.3855074998},"time":{"base_time":2340,"job_duration_modifier":null,"industry_skill_level":5,"advanced_industry_skill_level":5,"total_time_run":1527,"number_of_run_per_hour":2.357563850687623,"number_of_run_per_day":56.58153241650295},"volume":{"per_unit_volume":5.0,"per_run_volume":5.0,"per_hour_volume":11.787819253438114,"per_day_volume":282.9076620825147},"profit":{"markets_profit":[{"name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","profit_per_hour":-695135.6829253434,"profit_per_day":-16683256.39020824}]}},{"facility_name":"Maurasi - BPs Center, read BIO","costs":{"inputs":{"inputs":[{"name":"Oscillator Capacitor Unit","orders":[{"quantity":1,"price_per_unit":48191.09,"total":48191.09}]},{"name":"Morphite","orders":[{"quantity":1,"price_per_unit":43482.24,"total":43482.24}]},{"name":"Mechanical Parts","orders":[{"quantity":5,"price_per_unit":12060.16,"total":60300.8}]},{"name":"Plasma Pulse Generator","orders":[{"quantity":1,"price_per_unit":19833.91,"total":19833.91}]},{"name":"R.A.M.- Armor/Hull Tech","orders":[{"quantity":1,"price_per_unit":1060.36,"total":1060.36}]},{"name":"Construction Blocks","orders":[{"quantity":3,"price_per_unit":12334.49,"total":37003.47}]},{"name":"Fernite Carbide Composite Armor Plate","orders":[{"quantity":1,"price_per_unit":9372.52,"total":9372.52}]},{"name":"Quantum Microprocessor","orders":[{"quantity":1,"price_per_unit":65805.53,"total":65805.53}]},{"name":"Damage Control I","orders":[{"quantity":1,"price_per_unit":8207.74,"total":8207.74}]}],"total":293257.66},"job_cost":{"estimated_item_value":177538.88791922803,"system_cost_index":0.0751,"facility_tax":0.002,"total":13554.561475969304,"job_cost_modifier":0.03},"total_run":393327.6716342574,"total_per_unit":393327.6716342574},"time":{"base_time":2340,"job_duration_modifier":0.15,"industry_skill_level":5,"advanced_industry_skill_level":5,"total_time_run":1298,"number_of_run_per_hour":2.773497688751926,"number_of_run_per_day":66.56394453004623},"volume":{"per_unit_volume":5.0,"per_run_volume":5.0,"per_hour_volume":13.86748844375963,"per_day_volume":332.81972265023114},"profit":{"markets_profit":[{"name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","profit_per_hour":-750585.2217899282,"profit_per_day":-18014045.322958276}]}}]}
